examples/3.routes/9.amenities-on-route/README.md

```diff
@@ -0,0 +1,20 @@
+# Add amenities to your route
+
+I need coffee! The kids need a bathroom break! Feed me or I'll end it all! Common complaints heard whilst driving. With amenity-based-routing, your caffeine/bathroom/food/other needs can be catered too, all whilst you charge your vehicle.
+
```

Comment:
I loved this but I would make this a bit less extreme and more focus on a lunch stop.